package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "3box": "^1.19.1",
-    "@1hive/1hive-ui": "1.0.11",
+    "@1hive/1hive-ui": "^1.0.13",
```

Comment:
Let´s upgrade to the latest which includes the addition of arbiscan and polyscan explorers
```suggestion
    "@1hive/1hive-ui": "^1.0.14",
```